pkgs/nix-tools/default.nix

```
@@ -0,0 +1,7 @@
+{ darwin }:
```

Instead of passing a whole system to nix-tools it really only needs references to default path locations, this way we don't need a complete system to build the packages.

```suggestion
{ pkgs
, profile ? "/nix/var/nix/profiles/system"
, nixPackage ? "/nix/var/nix/profiles/default"
, defaultPath ? "/usr/local/bin:/usr/bin:/usr/sbin:/bin:/sbin"
}:

let
  extraPath = lib.makeBinPath [ nixPackage pkgs.coreutils pkgs.jq pkgs.git ];
in

{
  darwin-option = pkgs.writeProgram "darwin-option"
    {
      inherit profile;
      inherit (pkgs.stdenv) shell;
    }
    ./darwin-option.sh;

  darwin-rebuild = pkgs.writeProgram "darwin-rebuild"
    {
      inherit profile;
      inherit (pkgs.stdenv) shell;
      path = "${extraPath}:${defaultPath}";
    }
    ./darwin-rebuild.sh;
}
```

---

Updated :+1: